### PR TITLE
Fix npm registry settings showing for personal workspace

### PR DESCRIPTION
**Pull request by CompuIves**

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx [hunk 1]
```diff
@@ -57,10 +58,12 @@ export const SettingNavigation = (props: SettingsNavigationProps) => (
...
         label="Workspace"
       />
-      <NavigationLink
-        url={dashboardUrls.registrySettings(props.teamId)}
-        label="Npm Registry"
-      />
+      {!props.isPersonal && (
+        <NavigationLink
+          url={dashboardUrls.registrySettings(props.teamId)}
+          label="Npm Registry"
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx:64]: super silly pick, but 
```suggestion
          label="npm Registry"
```

**Comment by siddharthkp** [comment]
Related: We should put this in an IconButton to increase the click area

 + The tooltip text should be more informative? "Learn more about npm registry configuration"

image

**Review by siddharthkp — CHANGES_REQUESTED** [review]
Code looks good 👍  Added 2 feedback on UX.

**Comment by CompuIves** [comment]
> Related: We should put this in an IconButton to increase the click area
> 
> 
> 
> image
> 
> 
> 
> 
> 
> and The tooltip text should be more informative? "Learn more about npm registry configuration"

Good point! I'll change the content of the tooltip. For some reason the tooltip stopped working when I put it outside of it, I got a CodeSadbox and once I fixed that I couldn't see the tooltip. Pretty weird 🤨

File: packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx (hunk 1 above)
Comment (CompuIves) [reply packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx:64]: Oh yes I've been thinking about this! So npm needs to be written without capitals, but when it's at the start of a sentence, shouldn't then the first letter be capitalized like any other word? Not sure, I at least do like the capital more from a design perspective (consistency), but that shouldn't be the reason to do it.

Comment (garethx) [reply packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx:64]: That's technically right, yes, but the mixed case does look wrong here somehow. Try all upper case e.g. NPM Registry - that will visually look better and folks will still know what you mean.

**Comment by siddharthkp** [comment]
> For some reason the tooltip stopped working when I put it outside of it, I got a CodeSadbox 

strange! 🤔 

If you see it again, can you pass me a repro sandbox

File: packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx (hunk 1 above)
Comment (CompuIves) [inline packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx:64]:
```suggestion
          label="NPM Registry"
```

**Comment by CompuIves** [comment]
> > For some reason the tooltip stopped working when I put it outside of it, I got a CodeSadbox
> 
> strange! 🤔
> 
> If you see it again, can you pass me a repro sandbox

Will do!

**Pull request merged**